### PR TITLE
Trev8939/ColormapRenderer

### PR DESCRIPTION
**Pull request by TADraeseke**
Adds a sample demonstrating use of the ColormapRenderer

### CODE REVIEW

**Comment by TADraeseke** [comment]
@doneill For your review. As usual, sample requires use to download data locally to the device. Have left //TODO tags in the README for links. Thanks!

File: colormap-renderer/src/main/java/com/esri/arcgisruntime/sample/colormaprenderer/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+    mMapView.setMap(map);
+    // add the raster as an operational layer
+    map.getOperationalLayers().add(rasterLayer);
+    // create a color map where values 0-149 are red (0xFFFF0000) and 150-250 are yellow (0xFFFFFF00)
+    List<Integer> colors = new ArrayList<>();
+    for (int i = 0; i <= 250; i++) {
+      if (i < 150) {
+        colors.add(i, 0xFFFF0000);
+      } else {
+        colors.add(i, 0xFFFFFF00);
```
Comment (doneill) [inline colormap-renderer/src/main/java/com/esri/arcgisruntime/sample/colormaprenderer/MainActivity.java]: Replace sRGB values with Color constants, e.g. `Color.RED`, and update the comment as appropriate.

**Comment by doneill** [comment]
Updated data links as well.

File: colormap-renderer/src/main/java/com/esri/arcgisruntime/sample/colormaprenderer/MainActivity.java (hunk 1 above)
Comment (TADraeseke) [inline colormap-renderer/src/main/java/com/esri/arcgisruntime/sample/colormaprenderer/MainActivity.java]: Done!

**Pull request merged**